ts/components/ui/BannerErrorState.tsx

```diff
@@ -0,0 +1,230 @@
+import {
+  ButtonLink,
+  Icon,
+  IOBannerBigSpacing,
+  IOBannerRadius,
+  IOColors,
+  IOIcons,
+  IOScaleValues,
+  IOSpringValues,
+  IOStyles,
+  Label,
+  VSpacer,
+  WithTestID
+} from "@pagopa/io-app-design-system";
+import React, { useCallback } from "react";
+import {
+  AccessibilityRole,
+  GestureResponderEvent,
+  Pressable,
+  StyleSheet,
+  View,
+  ViewStyle
+} from "react-native";
+import Animated, {
+  Extrapolation,
+  FadeIn,
+  FadeOut,
+  interpolate,
+  useAnimatedStyle,
+  useDerivedValue,
+  useSharedValue,
+  withSpring
+} from "react-native-reanimated";
+
+/* Styles */
+const colorContent: IOColors = "grey-700";
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: "row",
+    alignItems: "center",
+    alignContent: "center",
+    borderRadius: IOBannerRadius,
+    borderCurve: "continuous",
+    padding: IOBannerBigSpacing
+  }
+});
+
+const dynamicContainerStyles = (
+  color: BaseBannerErrorStateProps["color"]
+): ViewStyle => ({
+  backgroundColor: IOColors[mapBackgroundColor[color]]
+});
+
+/* Component Types */
+
+type BaseBannerErrorStateProps = WithTestID<{
+  icon?: IOIcons;
+  label: string;
+  color: "neutral" | "turquoise";
+  viewRef?: React.RefObject<View>;
+  // A11y related props
+  accessibilityLabel?: string;
+  accessibilityHint?: string;
+}>;
+
+type BannerErrorStateActionProps =
+  | {
+      actionText?: string;
+      onPress: (event: GestureResponderEvent) => void;
+      accessibilityRole?: never;
+    }
+  | {
+      actionText?: never;
+      onPress?: never;
+      accessibilityRole?: AccessibilityRole;
+    };
+
+export type BannerErrorStateProps = BaseBannerErrorStateProps &
+  BannerErrorStateActionProps;
+
+// COMPONENT CONFIGURATION
+
+/* Used to generate automatically the colour variants
+in the Design System screen */
+export const bannerBackgroundColors: Array<BannerErrorStateProps["color"]> = [
+  "neutral",
+  "turquoise"
+];
+
+const mapBackgroundColor: Record<
+  NonNullable<BannerErrorStateProps["color"]>,
+  IOColors
+> = {
+  neutral: "grey-50",
+  turquoise: "turquoise-50"
+};
+
+export const BannerErrorState = ({
+  viewRef,
+  icon,
+  color,
+  label,
+  actionText,
+  onPress,
+  accessibilityHint,
+  accessibilityLabel,
+  testID
+}: BannerErrorStateProps) => {
+  const isPressed: Animated.SharedValue<number> = useSharedValue(0);
```

Comment:
Access to `SharedValue` from `Animated` is now deprecated. Just use `SharedValue`

```suggestion
  const isPressed: SharedValue<number> = useSharedValue(0);
```

Reply:
Thank you for pointing it out! Addressed into 878f0ffaba36f63de6ca058e3baa63936a4c43c2